<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
  <class>GUI_ChangePassword</class>
  <widget class="QDialog" name="GUI_ChangePassword">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>125</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>125</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>300</width>
        <height>125</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Смена пароля</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
            <widget class="QLabel" name="oldPasswordLabel">
              <property name="text">
                <string>Старый пароль:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="oldPasswordText">
              <property name="echoMode">
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="newPasswordLabel">
              <property name="text">
                <string>Новый пароль:</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLineEdit" name="newPasswordText">
              <property name="echoMode">
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="duplicatedPasswordLabel">
              <property name="text">
                <string>Повторите пароль:</string>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QLineEdit" name="duplicatedPasswordText">
              <property name="echoMode">
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="QDialogButtonBox" name="buttonBox">
              <property name="standardButtons">
                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <resources />
  <connections>
    <connection>
      <sender>buttonBox</sender>
      <signal>rejected()</signal>
      <receiver>GUI_ChangePassword</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>264</x>
          <y>106</y>
        </hint>
        <hint type="destinationlabel">
          <x>281</x>
          <y>243</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>